SEP-0002.md

```diff
@@ -138,15 +137,27 @@ ED shall serve a one year term, and can be re-elected.
 ## The Executive Director
 The role of the executive director is to manage the day to day
 operations of the SunPy organization. The executive director (ED for
-short) shall report to the board. The executive director shall have
-push privileges to the SunPy repository and can elect SunPy
-maintainers from the developer community to have the same privileges.
-The ED shall have ownership privileges to the sunpy github
-organization. The ED shall hold regular meetings with the developer
-community (developer meeting).
+short) shall:
+
+* hold regular meetings with the developer community (at least monthly)
+* make regular reports to the board (at least quarterly)
+* have push privileges to the SunPy repository
+and can delegate those privileges
+* have ownership privileges to the SunPy GitHub organization
+* create and maintain the SunPy code of conduct (see below)
 
 The current Executive Director is named in SEP-0006.
 
+### Code of Conduct
+
+The SunPy organization shall have an explicit code of conduct to state
+the norms of peer interactions, including inclusiveness and respect,
+and describe the process for dispute resolution.
+The code of conduct shall be publicly posted on the SunPy website.
+The code of conduct shall be accessible in such
+a manner as to welcome and incorporate feedback from the developer
+community (e.g., a Git respository).
```

Comment:
i would remove mention of a git repository as it is not needed to mention a potential solution here. Should also mention a anonymous way to provide feedback.